### ISSUE TITLE
403 Forbidden - After migrating to Frank version v9.0.0-20241124.042330 and tomcat 10

### ISSUE THREAD

**Issue by ramanjulu-decos** [frankframework/frankframework#7993]
### Current Behavior

![image](https://github.com/user-attachments/assets/0547c147-dd4a-46da-836f-3ba91ee12744)

### Expected Behavior

Should give us 200

### Environment Information

Local development environment

**Comment by nielsm5** [comment]
Do you have any form of authentication configured?

**Comment by ramanjulu-decos** [comment]
@nielsm5 Yes, it's Oauth2 configuration in place, which is using ADFS.

**Comment by nielsm5** [comment]
And you have a oauth-role-mapping.properties file to map roles/scopes to the application roles?
```properties
IbisObserver=SCOPE_openid
IbisAdmin=SCOPE_role_a
IbisDataAdmin=SCOPE_role_b
IbisTester=SCOPE_role_c
```
It's very peculiar that an endpoint which only requires 1 role to be present to return a 403.

**Comment by ramanjulu-decos** [comment]
@nielsm5  release 8.2 has that configuration file

![image](https://github.com/user-attachments/assets/97032d10-e9b0-4954-914f-49f7d85996da)

But master branch doesn't have it

![image](https://github.com/user-attachments/assets/a68da839-e58a-4522-a674-993e7f9bd166)

**Comment by nielsm5** [comment]
> @nielsm5  release 8.2 has that configuration file
> 
> 
> 
> ![image](https://github.com/user-attachments/assets/97032d10-e9b0-4954-914f-49f7d85996da)
> 
> 
> 
> But master branch doesn't have it
> 
> 
> 
> ![image](https://github.com/user-attachments/assets/a68da839-e58a-4522-a674-993e7f9bd166)
> 
> 

Hi Ram,

That's because it was an example file and gives logged in users all access by default. It was not possible to overwrite these settings in certain docker situations, which is why we've opted to remove it. 

You can add this file to your resource folder and change the role to scope mapping according to your preferences.

**Comment by nielsm5** [comment]
Hey!

Any update/progress on this? Did adding the file help at all?

**Comment by ramanjulu-decos** [comment]
It didn't work.

![image](https://github.com/user-attachments/assets/da4ab4d9-7e07-40ed-b177-badd54b54121)


I have tested below two use cases

1. Added the role mapping properties files, tested and it didn't work
2. Disabled Oauth configuration in the catalina.properties file, tested and it didn't work

**Comment by nielsm5** [comment]
Hi Ram,

We're unable to reproduce this. Is it just the flow images that are not working?
What happens when you set `application.security.console.authentication.type=NONE` ?

**Comment by ramanjulu-decos** [comment]
yes, I have used same property in catalina.properties file and tested

![image](https://github.com/user-attachments/assets/82cc6159-1535-4176-a595-242a5976700e)

![image](https://github.com/user-attachments/assets/144d9a93-61f5-4320-9b97-2b863d8915e3)



with ADFS(Oauth) used below configuration in the catalina.properties file

# Define the available authenticators
application.security.http.authenticators=myADAuth

# Configure the authenticators (type + properties)
application.security.http.authenticators.myADAuth.type=OAUTH2
application.security.http.authenticators.myADAuth.scopes=openid,profile,email
application.security.http.authenticators.myADAuth.authorizationUri=
application.security.http.authenticators.myADAuth.tokenUri=
application.security.http.authenticators.myADAuth.jwkSetUri=
application.security.http.authenticators.myADAuth.issuerUri=
application.security.http.authenticators.myADAuth.userInfoUri=
application.security.http.authenticators.myADAuth.userNameAttributeName=name
application.security.http.authenticators.myADAuth.clientId=
application.security.http.authenticators.myADAuth.clientSecret=
application.security.http.authenticators.myADAuth.provider=custom



# Set the autheticator on an endpoint
servlet.IAF-API.authenticator=myADAuth

servlet.Console.authenticator=myADAuth

**Comment by nielsm5** [comment]
Hi Ram,

We removed the console from the worker/agent, and as such it's now a Spring Boot application.

We changed the name of the property to it would become 1 instead of 2 separate properties. See

https://github.com/frankframework/frankframework/blob/0f304eaba88bafe6c33e621ef07ac0d621ad0801/console/war/src/main/resources/application.properties#L43-L45

We've also we have expanded [our manual](https://frank-manual.readthedocs.io/en/latest/deploying/IdentityProviders/MicrosoftEntraId/microsoftEntraId.html) in regards to this topic.

**Comment by ramanjulu-decos** [comment]
Hi @nielsm5 Now running into below issue

![image](https://github.com/user-attachments/assets/2e29bb86-075e-4713-9cf1-41928b5b7e75)

**Comment by nielsm5** [comment]
Hey Ram,

Apologies for the delay. I'm unsure why this is happening, especially on localhost. but for the time being, with `cspheader.reportOnly=true` you can disable the CSP header.

**Comment by nielsm5** [comment]
I hope /  assume it's working now? If you have any further questions please open a new issue :)
